ledgerblue/readElfMetadata.py

```diff
@@ -44,7 +44,7 @@ def _get_elf_file(filename):
 
 
 def _get_elf_section_value(elf, section_name):
-    section = elf.get_section_by_name(f"ledger.{section_name}")
+    section = elf.get_section_by_name(f".ledger.{section_name}")
```

Comment:
We should drop this change and merge the PR no?

Reply:
Yep will do

Reply:
Done @xchapron-ledger